tests/test_upstream_issue.py

```diff
@@ -564,7 +587,42 @@ def test_handle_github_message_no_comments(self, mock_issue_from_github, mock_gi
 
     @mock.patch(PATH + 'Github')
     @mock.patch('sync2jira.intermediary.Issue.from_github')
-    def test_handle_github_message_successful(self,
+    def test_handle_old_style_github_message_successful(self,
+                                              mock_issue_from_github,
+                                              mock_github):
+        """
+        This function tests 'handle_github_message' where everything goes smoothly!
+        """
+        # Set up return values
+        mock_issue_from_github.return_value = "Successful Call!"
+        mock_github.return_value = self.mock_github_client
+
+        # Call function
+        response = u.handle_github_message(
+            msg=self.old_style_mock_github_message,
+            config=self.mock_config
+        )
+
+        # Assert that calls were made correctly
+        mock_issue_from_github.assert_called_with('org/repo',
+                                                  {'labels': ['custom_tag'], 'number': 'mock_number',
+                                                   'comments': [{'body': 'mock_body', 'name': unittest.mock.ANY,
+                                                                 'author': 'mock_username', 'changed': None,
+                                                                 'date_created': 'mock_created_at', 'id': 'mock_id'}],
+                                                   'assignees': [{'fullname': 'mock_name'}],
+                                                   'filter1': 'filter1', 'user':
+                                                       {'login': 'mock_login', 'fullname': 'mock_name'},
+                                                   'milestone': 'mock_milestone'}, self.mock_config)
+        mock_github.assert_called_with('mock_token', retry=5)
+        self.assertEqual('Successful Call!', response)
+        self.mock_github_client.get_repo.assert_called_with('org/repo')
+        self.mock_github_repo.get_issue.assert_called_with(number='mock_number')
+        self.mock_github_issue.get_comments.assert_any_call()
+        self.mock_github_client.get_user.assert_called_with('mock_login')
+
```

Comment:
Rather than cookie-cuttering this code, I suggest reusing the code in `test_handle_new_style_github_message_successful()` with each of the old and new style of message, to avoid the code duplication.

Reply:
After a305ae3, this can be done more simply.

Reply:
Like in 2218859 ?